merge_test.go

```diff
@@ -231,3 +232,74 @@ func TestDB_Multi_Open_Merge(t *testing.T) {
 	}
 	assert.Equal(t, len(kvs), db.index.Size())
 }
+
+func TestDB_Merge_ReopenAfterDone(t *testing.T) {
+	options := DefaultOptions
+	db, err := Open(options)
+	assert.Nil(t, err)
+	defer destroyDB(db)
+
+	kvs := make(map[string][]byte)
+	for i := 0; i < 10000; i++ {
```

Comment:
The test data can be more large to generate more than one file.
Some problems may occur with many wal files.